### ISSUE TITLE
PDF report are getting crashed

### ISSUE THREAD

**Issue by ksraja00** [chaostoolkit/chaostoolkit-reporting#48]
Traceback (most recent call last):
  File "/usr/local/bin/chaos", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/chaosreport/cli.py", line 95, in report
    save_report(header, reports, report_path, export_format)
  File "/usr/local/lib/python3.9/site-packages/chaosreport/__init__.py", line 313, in save_report
    major, minor, _ = pandoc_version.split(".", 2)

**Comment by cdsre** [comment]
What version of pandoc do yo have installed? I have seen this before with later version of pandoc providing only the `major.minor` version of semver. The ctk-reporting code in its current form expects at minimum a semver version of `major.minor.patch`. 

```python
        pandoc_version = pypandoc.get_pandoc_version()
        major, minor, _ = pandoc_version.split(".", 2)
        if int(major) == 2 and int(minor) < 19:
            extra_args.append("--self-contained")
        else:
            extra_args.append("--embed-resources")
```

When i ran into this issue I had to downgrade pandoc to `3.2.1` to make it work  currently. I will also look at a fix to this in the code and submit a PR today if I get time.

**Comment by Lawouach** [comment]
This is now fixed and released in 0.18.0 thanks to @cdsre :)
